padertorch/contrib/data/wsj0_mix/prepare_data.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Put your paths here!
+
+# Path to the database as generated by the matlab scripts
+database_path=
+
+# If you need transcriptions, put the path to the WSJ root folder here
+wsj0_root=
```

Comment:
Can you adopt the idea from https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/db.sh ?
In this way, we can specify some defaults for us.

Reply:
Yes, what is the best hostname to check for? Is `[[ "$(hostname -d)" == "nt.uni-paderborn.de" ]];` enough, or shall we include the PC2 cluster as well?

Reply:
Yes, that is fine for the workstations. For the clusters I would suggest to check `$PC2SYSNAME` if it is `OCULUS`.

Reply:
Done